doc/src/guides/solving.md

```diff
@@ -0,0 +1,45 @@
+# Solve Using the SymPy Python Library
+
+The Python package SymPy can symbolically solve equations, differential equations, 
+linear equations, nonlinear equations, matrix problems, inequalities, 
+Diophantine equations, and evaluate integrals. SymPy can also solve numerically.
+
+Learn how to use SymPy computer algebra system to:
+
+| Description                                                  | Example                                                                                                                     |
+|--------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|
+| [ Solve an equation algebraically ](http://docs.sympy.org)                        | [  $x^2 = 4 \to x =[-2, 2] $  ](http://docs.sympy.org)                                                                                                       |
+| [ Solve a system of equations algebraically ](http://docs.sympy.org)              | [  $x^2 + y = 2, x - y = 4 \to [(x = -3, y = -7), (x = 2, y = 2)]$  ](http://docs.sympy.org)                                                                                              |
+| [ Solve an equation numerically ](http://docs.sympy.org)                          | [ $cos(x) = x \to x \approx 0.739085133215161$ ](http://docs.sympy.org)                                                                                                     |
+| [ Solve a partial differential equation algebraically ](http://docs.sympy.org)    | [${\frac {\partial ^{2}(x^3 y^3)}{\partial x\partial y}}=9 \to x \ne 0, y = \pm\frac{1}{x} $](http://docs.sympy.org)                                                                     |
```

Comment:
This isn't really a PDE. A PDE would have an unknown function of multiple variables, similar to an ODE. SymPy's PDE support isn't very good, so for an example you may need to copy one from the docs just to be sure that SymPy can actually solve it (`pdsolve`). 

Alternatively, we can just omit PDEs from here for now, since SymPy's symbolic PDE solving support is so weak, especially for the sorts of PDEs people might actually want to solve analytically.

Reply:
Thanks--this exceeds my level of math education :-) This is the example I found on the [`pdsolve` page](https://docs.sympy.org/dev/modules/solvers/pde.html?highlight=pdsolve#sympy.solvers.pde.pdsolve):

```
>>> from sympy.solvers.pde import pdsolve
>>> from sympy import Function, Eq
>>> from sympy.abc import x, y
>>> f = Function('f')
>>> u = f(x, y)
>>> ux = u.diff(x)
>>> uy = u.diff(y)
>>> eq = Eq(1 + (2*(ux/u)) + (3*(uy/u)), 0)
pdsolve(eq)
```

Is there a way to express this in one line of math, or at least fewer than four?

Reply:
Let's just nix the PDE section for now. If we want to include it we should take a look at stackoverflow and other sources to get an idea of common problems that people are trying to solve, and whether SymPy can actually solve them or not.

Reply:
OK, I'll remove for now. PDE is the [fourth most common category of solving type](http://www.sympy.org/sympy-search-analytics/search-terms.html) (16215 Google search impressions per year = 44 per day), so we should try to add it back in later. The [common types seem to be those used for physics applications (heat, wave, etc.)](https://en.wikipedia.org/wiki/Partial_differential_equation#See_also).

A specific example is [heat flow in a uniform rod](https://en.wikipedia.org/wiki/Heat_equation#Heat_flow_in_a_uniform_rod). If someone with more math expertise could formulate and solve that, or another common and simple example, in SymPy, I'd appreciate it and would add it to this landing page and a PDE sub-page.